ext/BijectorsReverseDiffExt.jl

```diff
@@ -253,11 +255,19 @@ end
 @grad_from_chainrules _transform_ordered(y::Union{TrackedVector,TrackedMatrix})
 @grad_from_chainrules _transform_inverse_ordered(x::Union{TrackedVector,TrackedMatrix})
 
-@grad_from_chainrules update_triu_from_vec(vals::TrackedVector{<:Real}, k::Int, dim::Int)
+@grad_from_chainrules Bijectors.update_triu_from_vec(
+    vals::TrackedVector{<:Real}, k::Int, dim::Int
+)
 
 @grad_from_chainrules _link_chol_lkj(x::TrackedMatrix)
 @grad_from_chainrules _inv_link_chol_lkj(x::TrackedVector)
 
+@grad_from_chainrules Bijectors.cholesky_lower(X::TrackedMatrix)
+@grad_from_chainrules Bijectors.cholesky_upper(X::TrackedMatrix)
+
+# TODO: Type-piracy; probably shouldn't do this.
```

Comment:
No, this should really not be defined in Bijectors. I'm sure this will lead to surprising debugging and issues when Bijectors is (not) loaded.